test/unit/units/post_mount_system_test.py

```diff
@@ -34,3 +45,22 @@ def test_main(
             call(['udevadm', 'trigger', '--type=subsystems', '--action=add']),
             call(['udevadm', 'trigger', '--type=devices', '--action=add'])
         ]
+
+    @patch.dict(os.environ, {'foo': 'bar', 'a': 'b'}, clear=True)
```

Comment:
The testing should also be setup to trigger an actual output of the `tolower()`, i.e. have the config with uppercase and then validate that lower case pops out at the end so we get the whole chain of the implementation to do something.